core/tests/test_utils.py

```diff
@@ -529,6 +625,23 @@ def save_new_valid_collection(
             language_code=constants.DEFAULT_LANGUAGE_CODE,
             exploration_id='an_exploration_id',
             end_state_name=DEFAULT_END_STATE_NAME):
+        """Creates an Oppia collection and adds a node saving the
+        exploration details.
+
+        Args:
+            collection_id: str. Id for the collection to be created.
```

Comment:
Id --> The ID (or id, depending on which choice you make for this -- see above).

Reply:
Done!